install_config/aggregate_logging.adoc

```diff
@@ -398,28 +387,34 @@ endif::openshift-enterprise[]
 
 == Post-deployment Configuration
 
+[[aggregated-elasticsearch]]
 === Elasticsearch
 
-In any production employment, Elasticsearch should be deployed with a
-cluster size of at least three for resiliency to node failures. Each instance
-requires individual storage, but an OpenShift deployment can only provide
-volumes shared by all its pods, so the Elasticsearch cluster cannot be
-implemented with a single deployment. The EFK deployer instead creates one
-deployment per instance. To view all current deployments of Elasticsearch:
+In a production environment, Elasticsearch should be deployed with a cluster
+size of at least three hosts to account for resiliency. Each Elasticsearch
```

Comment:
Changed to:

"A highly-available environment requires at least three replicas of
Elasticsearch; each on a different host."

I feel it encompasses what you're saying without putting down the bits that you don't like.